### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by Sandeep418** [rohit-gupta/Video2Language#12]
I am having a problem with the prediction I have trained the model for 100 epochs but still, I got 50% accuracy and I have also tried this model to predict on the training set but the prediction accuracy is very very poor.

**Comment by rohit-gupta** [comment]
Hi @SandeepRddy, accuracy is not a good metric to judge a captioning model, please use the METEOR score.
